apps/judicial-system/api/infra/judicial-system-api.ts

```diff
@@ -44,8 +44,8 @@ export const serviceSetup = (services: {
       },
       HIDDEN_FEATURES: {
         dev: '',
-        staging: '',
-        prod: '',
+        staging: 'POLICE_CASE_FILES',
```

Comment:
You can now use https://docs.devland.is/technical-overview/devops/service-setup#server-side-feature-flags maybe?